SETUP.md

```diff
@@ -414,27 +391,7 @@ Wondering where to start?  See our [contribution guidelines](CONTRIBUTING.md) fo
 ---
 ### Bundle Install Tips
 
-#### Apple Silicon (M1) or Intel Mac bundle install steps
-
-On Apple Silicon/Intel Mac, additional steps are required to get `bundle install` to work.
-
-First, run the following commands to successfully complete a bundle install:
-
-```sh
-gem install bundler -v 2.3.22
-rbenv rehash
-export LIBRARY_PATH=$LIBRARY_PATH:/opt/homebrew/opt/openssl/lib/
-bundle install
-```
-
-After `bundle install` completes successfully, before attempting the `bundle exec rake ...` commands, execute the following command to avoid errors with **ffi**
-
-```sh
-bundle update ffi
-```
-
-
-### ImageMagick with Pango
```

Comment:
I started fresh, cleared my ~/.rbenv, and I'm pretty sure this is no longer necessary. Brew was messing up the openssl include and library path before (couple years past), but its straightened out now. Explicitly updating ffi is no longer required, our Gemfile.lock version is plenty fresh.